modules/reference/pages/command/featureUtility-installFeature.adoc

```diff
@@ -33,6 +33,12 @@ Install the MicroProfile Health 2.2 feature without caching the feature to the l
 featureUtility installFeature mpHealth-2.2 --noCache
 ----
 
+Install the `userFeature-1.0` custom user feature from a Maven repository:
+
+----
+featureUtility installFeature userFeature-1.0 --featuresBom=my.user.features:myBOM:<runtime-version>
+----
+
```

Comment:
for both the usage examples- italicize runtime-vesrion and remove the brackets:

featureUtility installFeature userFeature-1.0 --featuresBom=my.user.features:myBOM:_runtime-version_

where runtime-version is the version of the Open Liberty server where you want to install the feature, for example, 21.0.0.11.

you'll have to add the [subs=+quotes] format to thew codeblocks to get them to render properly